mapo/agents/mapo/tests/test_exploration_state.py

```diff
@@ -22,6 +23,7 @@ def test_deterministic_evaluation():
     assert np.allclose(action1, action2)
 
 
+@pytest.mark.skip
```

Comment:
Why?

Reply:
I think I explained it in this [commit](bc5f6417c96240b6d92503bfff54e8dcd6c05e9f).

Reply:
I could maybe try to increase the batch size.

Reply:
Indeed, you could try to increase the batch size and if this slows down the test too much consider decorate it with `pytest.mark.slow` ...

Reply:
Actually, I think I made a grave error... I'm testing if the output distribution of the policy is a gaussian with the mean in the deterministic action and standard deviation defined by the config. The problem is that I clip the actions in `build_action_sampler` to the desired action space. No wonder the test fails, even when (and specially when) I increase the batch size. 

I'll make sure to use an unbounded action space for this test.